### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by iharshit009**
## Issue that this pull request solves

Closes: #247 

## Proposed changes
Updated the README with correction

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (please describe): 
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [`x`] Documentation update (Documentation content changed)

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by iharshit009** [comment]
HI @rpotter12 @xandao6 I just changes a word in README, Could you please tell me why am I getting these checks unsuccessful?

**Comment by priyansh19** [comment]
@iharshit009 for CircleCI it's not your fault.

**Comment by rpotter12** [comment]
@iharshit009 please change the readme properly. New options are added in the software.
1. Add new pic of usage section. 
2. Pull command is not used as it is written. Please se the wiki pages of the repository. There you will find the exact command.

File: wplay/changeAbout.py [hunk 1]
```diff
@@ -4,9 +4,11 @@
 import sys
 import requests
 
+
 async def changeAbout():
     page, _ = await browser_config.configure_browser_and_load_whatsapp()
-    choice = input("Choose 1 for changing About section every second and 2 for changing About in every 30 min : ")
+    choice = input(
+        "Choose 1 for changing About section every second and 2 for changing About in every 30 min : ")
```
Comment (rpotter12) [inline wplay/changeAbout.py]: why you have used next line??

Comment (xandao6) [reply wplay/changeAbout.py]: actually isn't wrong, maybe was the auto-formatter

Comment (iharshit009) [reply wplay/changeAbout.py]: @rpotter12 I hadn't made any changes to this deliberately
This is due to auto-formatter

**Comment by iharshit009** [comment]
@xandao6 @rpotter12 does it need anymore changes?

**Pull request merged**